r_markdown/emissions.Rmd

```diff
@@ -196,21 +320,20 @@ ___
 
 * account for artifact
 
+Use replicate average artifact (probably clean up spreadsheet at some point)
```

Comment:
umm, we had a long discussion about this back in the day when I didn't capitulate so easily?